src/commands/zrange.json

```
@@ -44,11 +44,11 @@
                 "key_spec_index": 0
             },
             {
-                "name": "min",
+                "name": "start",
                 "type": "string"
             },
             {
-                "name": "max",
+                "name": "stop",
```

```suggestion
                "name": "end",
```

in the code the variables are named:
```c
    long opt_start = 0;
    long opt_end = 0;
```
so maybe using `end` is slightly better than `stop`?

---

@oranagra honestly the reason for `stop` over `end` is because that's what the now deprecated [`ZREVRANGE`](https://redis.io/commands/zrevrange) command uses as its argument names. Can go either way - whichever you prefer.

---

i see.. i agree we can go either way.
we can also decide to rename the arguments of ZREVRANGE*.
@itamarhaber i leave it for you to decide.